lib/data-types.js

```diff
@@ -417,6 +417,21 @@ class TIME extends ABSTRACT {
   toSql() {
     return 'TIME';
   }
+  _stringify(date) {
+    if (date instanceof Date) {
+      return moment(date).format('HH:mm:ss');
+    } 
+    return date;
+  }
+  _sanitize(value, options) {
+    if ((!options || options && !options.raw) && value instanceof Date ) {
+      return moment(value).format('HH:mm:ss');
+    }
+    return value;
+  }
+  _isChanged(value, originalValue) {
+    return value !== originalValue;
```

Comment:
this is going to break if `value` and `originalValue` are both instances of the JS `Date` class, since this compares the references not the values.

eg, `new Date() !== new Date()`. 

`_isChanged` in `DATE` has logic to take care of this, you could probably copy and paste that.

Reply:
I have changed the PR  name to match the convention, also has updated `__isChanged` function with code from `DATE` class. Hope the rest looks good.